### PR TITLE
Add functional testing cherry

### PR DESCRIPTION
**Pull request by ALonelyTower**
### Issue Reference
Pull Request is in reference to #63 

### Summarize
Add browser interaction tests to repository.
1.  Update requirement.txt to include Selenium.
2.  Update README.md to inform of new dependency (geckodriver)
3.  Modify .travis.yaml to include needed dependencies for Selenium
4.  Add functional test to verify clicking Request Help link navigates to Request Help page.

### CODE REVIEW

**Comment by shajith** [comment]
@ALonelyTower i resolved conflicts on this. Are these tests running on travis now?

**Comment by ALonelyTower** [comment]
@shajith I don't suppose there's a way to rerun Travis CI?  Seems there's a conflict with the database migrations.

**Comment by ALonelyTower** [comment]
@shajith Alright, it's the test on my Fork have passed Travis.  If someone with write access could rerun the test for this pull, that'd be great.

**Comment by ALonelyTower** [comment]
Closed and opened the Pull Request to initiate the build process.

**Comment by ALonelyTower** [comment]
Resolved a merge conflict:  I stated an explicit version for python-dateutil, and assumed that it's the most recent version.

**Comment by ALonelyTower** [comment]
The failed test is the result of the Home Page - Request Help link text being changed in one of the commits of the merge, so the test is behaving as expected.

There's a discrepancy between the Malayalam text on the Home Page and the Request Help page; I won't refactor my test until we settle for one text or the other (Issue #934).

File: requirements.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ pillow==5.2.0
 boto3==1.7.80
 botocore==1.10.80
 django-storages==1.6.6
-python-dateutil
```
Comment (biswaz) [inline requirements.txt]: We use dateutil at 2 places:
1. views.py -- line 31
2. sms.py -- line 18
Please explain why do you want to replace it with python-dateutil

**Comment by biswaz** [comment]
Build passing now

**Comment by biswaz** [comment]
We're made changes based on your issue and added some more tests. Rebase now please

**Comment by ALonelyTower** [comment]
@biswaz I've done a rebase on a copy of the branch, but there are still failing unit tests.  Should I continue with my rebase?

current_test_state

**Comment by ALonelyTower** [comment]
Apologies for that last merge, first time doing a Pull Request, and I forgot that I allowed edits to be made.  Neglected to do a pull before the rebase.  If it's too unseemly, I can try to tidy it up by making a new branch / PR.

**Comment by biswaz** [comment]
A new pr will be better IMO. What do you say @ja8zyjits @ashwinrajeev

**Comment by ALonelyTower** [comment]
@biswaz Have the commits in this branch made by the others been applied to master?  Or will I need to save these commits as well?

**Pull request closed without merging**